tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -302,15 +306,17 @@ def reduction_setup():
     )  # type: ignore
 
 
-def test_maxover_execution_sparse(reduction_setup):
+def test_maxover_execution_sparse(reduction_setup, fieldview_backend):
     """Testing max_over functionality."""
+    if fieldview_backend == gtfn_cpu.run_gtfn:
+        pytest.skip(reason="not yet supported.")
```

Comment:
Looks like kwarg is wrong here. Needs to be fixed in more places (but if you do find/replace, don't break `@pytest.mark.skip(reason=` where it is correct.
```suggestion
        pytest.skip("not yet supported.")
```